compiler/qsc_project/src/tests.rs

```diff
@@ -436,7 +440,12 @@ fn explicit_files_list() {
                     packages: {},
                 },
                 lints: [],
-                errors: [],
+                errors: [
```

Comment:
nitpick: I'd prefer duplicating the original test, leaving it as the "confirm no errors in good configuration" test and making the copy a test specific to this new error.